src/components/PhotoAndText.js

```diff
@@ -0,0 +1,67 @@
+import { Row, Col, Typography } from "antd";
+import useWindowDimensions from "../hooks/useWindowDimensions";
+
+const { Paragraph } = Typography;
+
+const PhotoAndText = ({
+  imgSrc,
+  headingText,
+  descriptionText,
+  imgPlacement,
+}) => {
+  const { width } = useWindowDimensions();
+
+  return (
+    <div style={{ padding: "1rem " }}>
+      <Row gutter={16}>
+        {(imgPlacement === "left" || width < 768) && (
```

Comment:
A conditional statement to render the image based on it's placement in the json data and the screen size